### ISSUE TITLE
Index settings not imported if using `map`

### ISSUE THREAD

**Issue by jfletcher-nuxeo** [jprante/elasticsearch-knapsack#89]
I'm using knapsack to migrate and index from one name to another in the same instance. To be clear I'm using 1.5.2.2 but I don't think it matters for this.

Steps:

* Export an index "foo".
* Import it with `map`; `map=\{"foo":"new-foo"\}`.

Result: the index settings are lost; the index "new-foo" has default settings.

Workaround: export with `map`. Import as normal. The new index will include the settings.

The workaround is totally appropriate but I had to stumble upon it. If this is a not a bug, maybe just a readme update in the import section would be sufficient.

**Comment by jprante** [comment]
Yes, this is kind of a glitch. I have to add a check that index settings of mapped index names must be recognized when importing. An export with map points to the correct index name.

Sorry for the inconvenience.
